presto-main/src/main/java/com/facebook/presto/sql/planner/plan/PlanNode.java

```diff
@@ -66,4 +75,15 @@ public PlanNodeId getId()
     {
         return visitor.visitPlan(this, context);
     }
+
+    protected void validateOutputVariables()
+    {
+        List<VariableReferenceExpression> outputVariables = getOutputVariables();
+        if (outputVariables.isEmpty()) {
+            return;
+        }
+        List<String> outputSymbolNames = getOutputSymbols().stream().map(Symbol::getName).collect(toImmutableList());
```

Comment:
nit: We usually have separate lines for stream:
```java
   getOutputSymbols().stream()
           .map(Symbol::getName)
           .collect(toImmutableList());
```

But given the size of the PR, we can also adjust them later

Reply:
The rule for this was never clear to me. I took it as if it fits in one line then use oneliner, otherwise break after `stream()`. This line will eventually disappear by the end of this PR so I think it's ok to not be too picky about it. 😛